cypress/integration/frontpageLayoutSpec.js

```diff
@@ -0,0 +1,146 @@
+import services from '../support/worldServices';
+import { el } from '../support';
+import describeForLocalOnly from '../support/describeForLocalOnly';
+
+Object.keys(services).forEach(index => {
+  const xservice = services[index];
+  const service = index;
+
+  // TODO this needs to become a normal describe block once Mozart routing is set up
+  describeForLocalOnly(`frontpage tests for ${service}`, () => {
+    // eslint-disable-next-line no-undef
+    before(() => {
+      cy.visit(xservice.url);
+    });
+
+    describe('checks the components are present', () => {
+      // eslint-disable-next-line no-undef
+      before(() => {
+        cy.viewport(1008, 768);
+      });
+
+      describe('cookie banner', () => {
```

Comment:
Is this test not covered by `cookieBannerWS.js`?

Reply:
Yeah, I saw that was only merged in yesterday so I missed it.